pkgs/by-name/rm/rmg/package.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wrapQtAppsHook
+, boost
+, cmake
+, discord-rpc
+, freetype
+, hidapi
+, libpng
+, libsamplerate
+, minizip
+, nasm
+, pkg-config
+, qtbase
+, qtsvg
+, SDL2
+, speexdsp
+, vulkan-headers
+, vulkan-loader
+, xdg-user-dirs
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rmg";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "Rosalie241";
+    repo = "RMG";
+    rev = "v${version}";
+    hash = "sha256-SAQJKfYoouJ2DLVks6oXiyiOI2/kgmyaHqt/FRfqKjI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    nasm
+    pkg-config
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    boost
+    discord-rpc
+    freetype
+    hidapi
+    libpng
+    libsamplerate
+    minizip
+    qtbase
+    qtsvg
+    SDL2
+    speexdsp
+    vulkan-headers
+    vulkan-loader
+    which
```

Comment:
This doesn't exsit. `which` is usually a shell built in.

Reply:
`mupen64plus-core` seems to have a dependency on GNU `which`. When I was testing the package using `nix-build` on my system, I couldn't build `mupen64plus-core` without `which` as a buildInput. I've moved it over to nativeBuildInput, which I think is more suitable.

Reply:
@slam-bert Did you build this locally on your machine? For example with `nix-build -A rmg` from inside your nixpkgs directory.

Reply:
@bachp That's what I did. I was able to declare `which` as a dependency and a `nativeBuildInput` just fine.

Reply:
Did you check everything in corretly, as you can see from CI this still doesn't evaluate properly.

Reply:
I'm not sure if I did or not. I saw that the `mupen64plus` nixpkgs already has uses `which` as a `buildInput`, so the errors I've been reading from `ofborg` are perplexing to me.